contracts/actions/checkers/AaveV3RatioCheck.sol

```diff
@@ -65,7 +65,7 @@ contract AaveV3RatioCheck is ActionBase, AaveV3RatioHelper {
 
             // can't boost too much under targetRatio so we don't trigger repay after
             if (currRatio < (targetRatio - RATIO_OFFSET)) {
-                revert BadAfterRatio(startRatio, currRatio);
+                revert BadAfterRatio(targetRatio, currRatio);
```

Comment:
this is changed but everywhere in deployed contracts it's left (startRatio, currRatio). Would either redeploy all contracts again or just remove this change as it's not breaking or anything

Reply:
Given that this is a non breaking change, I would keep it as is and redeploy other versions when needed